### PR TITLE
Add geo redirection

### PR DESCRIPTION
**Pull request by palmenhq**
[here](https://lite.ip2location.com/norway-ip-address-ranges) is a list of norwegian IPs, for example `2.148.0.1`. You can test it by applying the `x-forwarded-for` header on localhost, for example `curl -vLo /dev/null -H 'x-forwarded-for: 8.8.8.8' localhost:8030` (8.8.8.8 is USA and v=verbose L=follow redirects o=dump response body here) should give an output similar to this: (i've highlighted the interesting stuff)

<pre>
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 8030 (#0)
<strong>> GET / HTTP/1.1</strong>
> Host: localhost:8030
> User-Agent: curl/7.64.1
> Accept: */*
> <strong>x-forwarded-for: 8.8.8.8</strong>
>
<strong>< HTTP/1.1 302 Found</strong>
< Vary: Accept-Encoding
<strong>< Location: /se-en</strong>
< Content-Type: text/html; charset=utf-8
< Content-Length: 57
< Date: Fri, 12 Jun 2020 09:00:22 GMT
< Connection: keep-alive
<
* Ignoring the response-body
{ [57 bytes data]
100    57  100    57    0     0   3166      0 --:--:-- --:--:-- --:--:--  3166
* Connection #0 to host localhost left intact
<strong>* Issue another request to this URL: 'http://localhost:8030/se-en'</strong>
* Found bundle for host localhost: 0x7fd80241c2c0 [can pipeline]
* Could pipeline, but not asked to!
* Re-using existing connection! (#0) with host localhost
* Connected to localhost (::1) port 8030 (#0)
<strong>> GET /se-en HTTP/1.1</strong>
> Host: localhost:8030
> User-Agent: curl/7.64.1
> Accept: */*
> <strong>x-forwarded-for: 8.8.8.8</strong>
>
<strong>< HTTP/1.1 200 OK</strong>
< Vary: Accept-Encoding
< Content-Type: text/html; charset=utf-8
< Content-Length: 2375109
< Date: Fri, 12 Jun 2020 09:00:22 GMT
< Connection: keep-alive
<
</pre>

### CODE REVIEW

**Review by vonElfvin — DISMISSED** [review]
Wow!

**Comment by gustaveen** [comment]
How do I apply headers in our setup @palmenhq :)

**Comment by gustaveen** [comment]
Awesome! 🎉   Didn't figure out how to get it working locally though :P

File: src/server/middlewares/redirects.ts [hunk 1]
```diff
@@ -18,3 +20,21 @@ export const forceHost = ({
 
   await next()
 }
+
+export const startPageRedirect: IMiddleware<object> = async (ctx) => {
+  const actualIp = ctx.ip
+  const country = lookupCountry(actualIp)
+  const queryStringMaybe = ctx.querystring ? '?' + ctx.querystring : ''
+
+  if (!country || country === 'SE' || country === 'FI') {
```
Comment (vonElfvin) [inline src/server/middlewares/redirects.ts:29]: FI?

Comment (vonElfvin) [reply src/server/middlewares/redirects.ts:29]: The Finnish are best in the world at english in school AFAIK. I'm not sure if their Swedish is better than their English.

Comment (vonElfvin) [reply src/server/middlewares/redirects.ts:29]: I not know the praxis for redirecting Finnish people on Swedish websites however 🤔

**Review by vonElfvin — APPROVED** [review]
Super clean!

File: src/server/middlewares/redirects.ts (hunk 1 above)
Comment (palmenhq) [reply src/server/middlewares/redirects.ts:29]: I have no clue tbh haha. It’s easy to switch language so maybe this is fine idk

**Pull request merged**